### DISCUSSION TITLE
WARNING: LLM.original_IS_CHANGED() got an unexpected keyword argument 'system_prompt'

### DISCUSSION THREAD

**Discussion by Jonseed** [heshengtao/comfyui_LLM_party#113] [Q&A]
I keep seeing this in the terminal. What does it mean?

**Comment by tone-milazzo** [comment]
Well, in my case it means the assistant_response output from the API LLM general Link node is always "Expecting value: line 1 column 1 (char 0)" Are you still getting results?

**Reply by heshengtao** [reply]
You can try fixing node.

**Comment by heshengtao** [comment]
This is just a meaningless alarm. The assistant replied with an error message. It must be something else. This error is usually parsed by treating the empty string as json. I just tested the latest version of party, and there is no problem. If you have any questions, you can tell me how your workflow is connected, what system it is, and what integration package it uses.

**Comment by osfa** [comment]
not completely meaningless, it seems to indicate that comfyui and the custom node repo are not in sync. this error leads to (for me at least) the workflow embedded in the output png not being able to be loaded in comfyui. as per: https://github.com/ltdrdata/ComfyUI-Inspire-Pack/issues/180 I updated this node and this warning went away and everything now works as expected.
